src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/ClassUseWriter.java

```diff
@@ -418,14 +420,19 @@ protected HtmlTree getClassUseHeader() {
 
     @Override
     protected Navigation getNavBar(PageMode pageMode, Element element) {
-        Content mdleLinkContent = getModuleLink(utils.elementUtils.getModuleOf(typeElement),
-                contents.moduleLabel);
-        Content classLinkContent = getLink(new HtmlLinkInfo(
-                configuration, HtmlLinkInfo.Kind.PLAIN, typeElement)
-                .label(resources.getText("doclet.Class"))
-                .skipPreview(true));
-        return super.getNavBar(pageMode, element)
-                .setNavLinkModule(mdleLinkContent)
-                .setNavLinkClass(classLinkContent);
+        List<Content> subnavLinks = new ArrayList<>();
+        if (configuration.showModules) {
+            ModuleElement mdle = configuration.docEnv.getElementUtils().getModuleOf(typeElement);
```

Comment:
It's more common to use `utils.elementUtils` than the expression `configuration.docEnv.getElementUtils()`